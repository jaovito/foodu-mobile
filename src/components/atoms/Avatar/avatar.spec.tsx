import {render} from '@test-utils';
import {Avatar} from './';
import defaultImage from '../../../assets/pizza.png';

describe('<Avatar /> tests', () => {
  it('should be defined', () => {
    const image = render(<Avatar source={defaultImage} />);

    expect(image).toBeDefined();
  });

  it('should be able to match with snapshot', () => {
    const image = render(<Avatar source={defaultImage} />).toJSON();

    expect(image).toMatchSnapshot();
  });
});
