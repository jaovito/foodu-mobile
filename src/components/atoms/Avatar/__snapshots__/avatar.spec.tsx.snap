// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Avatar /> tests should be able to match with snapshot 1`] = `
<Image
  onError={[Function]}
  resizeMode="cover"
  source={
    {
      "testUri": "../../../src/assets/pizza.png",
    }
  }
  style={
    [
      {
        "borderColor": "#E83F5B",
        "borderRadius": 125,
        "borderWidth": 6,
        "height": 157,
        "width": 157,
      },
    ]
  }
/>
`;
